cmd/ctl/main.go

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright 2020 The Jetstack cert-manager contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+
+	genericapiserver "k8s.io/apiserver/pkg/server"
+)
+
+func main() {
+	stopCh := genericapiserver.SetupSignalHandler()
```

Comment:
nit (one to follow up on as there's a few places that call this) - given the generic nature of the `pkg/server` package, I'm concerned that this probably adds some extra bloat to all our binaries when it is pulled in. This will be especially relevant with a CLI tool as users will be downloading this to their own machines. Perhaps we should have our own version of this in the repo 😄